frontend/index.html

```diff
@@ -25,6 +25,10 @@ <h1>Comunidade de jardinagem</h1>
                 </button>
                 <button class="botao-custom">
                     <a href="fav_plants.html">Plantas favoritas</a>
+
+                </button>
+                <button class="botao-header">
```

Comment:
This class changed from `botao-header` to `botao-custom` in #6, see below. In other pages the class is correct.

![image](https://user-images.githubusercontent.com/28306525/124460134-7f856b00-dd7e-11eb-9d08-cd062fa2098d.png)